plugin/virustotal/vtgrep.py

```diff
@@ -119,6 +123,11 @@ def pack(self):
         self.wcs_stream += '?' * (wcs_len % 2)
         self.packed = True
 
+  def same_type(self, next_slice):
+    if isinstance(next_slice, Bytes):
```

Comment:
Here, should be Wilcards?

Reply:
Not really, if the if statement is True then returns False